cfgov/v1/jinja2/v1/header.html

```diff
@@ -13,7 +13,9 @@
 {# Skip Link to skip to main content #}
 {# TODO: When projects standardize on #main,
    remove this and empty after-header div #}
-<a href="#after-nav" id="skip-nav">Skip to main content</a>
+<div class="skip-nav">
+    <a href="#after-nav" class="skip-nav_link">Skip to main content</a>
+</div>
```

Comment:
We should probably pull this out into its own include file so that we're not duplicating this markup here and in `base.html`, but it's fine for now.

Reply:
Agreed. For futureland